awesome.c

```diff
@@ -543,6 +543,9 @@ main(int argc, char **argv)
     /* We have no clue where the input focus is right now */
     globalconf.focus.need_update = true;
 
+    /* set the default preferred icon size */
+    globalconf.preferred_icon_size = 0;
```

Comment:
This doesn't really depend on the DPI. It depends on the size at which the icons will be visible. As a first approximation, this is the height of the wibox, but I guess that only works with the default config. (Plus, different screens can have different heights here)

If you want, we could add something to the default config which sets this to... something. But what?